### ISSUE TITLE
The sdcard is always busy after call refreshFileList

### ISSUE THREAD

**Issue by daoxuanquan** [dji-sdk/Mobile-SDK-iOS#536]
The sdcard was always busy after i called DemoComponentHelper.fetchCamera()?.mediaManager?.refreshFileList. I have tried to exit the playback mode by called exitPlayback() method and set the mode to shootPhoto but it seem not helpful. Because the sd card was busy so it unable to shoot photo either. I also tried on DJISdkDemo and got the same issues.
The steps for reproduce.
call method refreshFileList -> call shoot photo. (or check the sdcard by calling remainingSpaceInMB in DJICameraStorageState) -> got messages "Captured image failed because the sdcard is busy or the current mode not supported"
If you know how to take a picture affter calling refreshFileList, please point me out.

**Comment by DJI-William** [comment]
1. Call enterPlayback to change the camera mode to the media download mode.
2. After calling the refreshFileListWithCompletion, you need to check the DJIMediaFileListState. It should be DJIMediaFileListStateUpToDate. 
3. Call exitPlayback to exit the media download mode.
4. Call setFlatMode:withCompletion to change your camera mode to DJIFlatCameraModePhotoSingle and take photos.

**Comment by daoxuanquan** [comment]
![IMG_1614](https://user-images.githubusercontent.com/60870919/190339200-a23ef1b7-7d03-495e-afa6-3ad0fd79d5c4.png)
Yes. After i did like above steps. I got this message.

**Comment by DJI-William** [comment]
Which aircraft and camera you are using? Have you set the storage location correctly?

**Comment by daoxuanquan** [comment]
I'm using mavic air 2. I seem like only happened when have a lot of image in sd card > 200.

**Comment by daoxuanquan** [comment]
> 

I think so. Because everything is Ok if i not call refreshFileListWithCompletion

**Comment by daoxuanquan** [comment]
Seem the aircraft has a configuration about lock read and write ability of sd card. I have to restart the aircraft to put the state to normal.

![IMG_0765](https://user-images.githubusercontent.com/60870919/190371699-5951d76f-2b7d-40c0-bc07-cdfc84b9a4c7.png)

**Comment by DJI-William** [comment]
You are correct, it seems like the Mavic Air 2 has this kind of feature. Please send an email to support@dji.com asking how to disable this feature.
